src/internal_macros.rs

```diff
@@ -444,7 +444,10 @@ macro_rules! impl_bytes_newtype {
                             if let Ok(hex) = ::core::str::from_utf8(v) {
                                 $crate::hashes::hex::FromHex::from_hex(hex).map_err(E::custom)
                             } else {
-                                return Err(E::invalid_value($crate::serde::de::Unexpected::Bytes(v), &self));
+                                return Err(E::invalid_value(
+                                    $crate::serde::de::Unexpected::Bytes(v),
```

Comment:
How about `use $crate::serde::de::Unexpected;` so that we neither have long lines nor split lines?

Reply:
Good idea, done as suggested. Thanks